src/main/java/gregtech/loaders/recipe/chemistry/PolymerRecipes.java

```diff
@@ -471,4 +470,39 @@ private static void polybenzimidazoleProcess() {
                 .output(dust, ChromiumTrioxide, 4)
                 .buildAndRegister();
     }
+
+    public static void polycaprolactamProcess() {
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[HV]).duration(400)
+                .fluidInputs(Benzene.getFluid(1000))
+                .fluidInputs(Hydrogen.getFluid(6000))
+                .fluidOutputs(Cyclohexane.getFluid(1000))
+                .notConsumable(dust, Nickel)
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[LV]).duration(100)
+                .fluidInputs(Chlorine.getFluid(1000))
+                .fluidInputs(NitricOxide.getFluid(1000))
+                .fluidOutputs(NitrosylChloride.getFluid(1000))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[MV]).duration(100)
+                .fluidInputs(Cyclohexane.getFluid(1000))
+                .fluidInputs(NitrosylChloride.getFluid(1000))
+                .output(dust, CyclohexanoneOxime, 19)
+                .fluidOutputs(HydrochloricAcid.getFluid(1000))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().EUt(VA[HV]).duration(200)
+                .input(dust, CyclohexanoneOxime, 19)
+                .fluidInputs(SulfuricAcid.getFluid(1000))
```

Comment:
This recipe currently voids Sulfuric Acid with no return. It needs to give 1000mB dilute.